File: Quarta/SempliciAlgoritmi/Gauss/Gauss.cs
using System;

namespace Gauss
{
    class Program
    {
        static void Main(string[] args)
        {
            int numLetto;
            int somma;

            Console.WriteLine("Inserisci il numero : ");
            numLetto = Convert.ToDouble(Console.WriteLine(numLetto));

            somma = numLetto * (numLetto + 1) / 2;

            Console.WriteLine("la somma e' : " + somma);
        }
    }
}

File: Quarta/SempliciAlgoritmi/Gauss/Gauss.cpp
#include <iostream>
using namespace std;
int main()
{
    int numLetto;
    int somma;

    cout << "Inserisci un numero \t:";
    cin >> numLetto;

    somma = numLetto * (numLetto + 1) / 2;
    cout << "la somma e' \t: " << somma << endl;

    return 0;
}